api/types.go

```diff
@@ -61,6 +61,19 @@ type DeleteRequest struct {
 	Name string `json:"name"`
 }
 
+type ShowRequest struct {
+	Name string `json:"name"`
+	Type string `json:"type"`
+}
+
+type ShowResponse struct {
+	License    string `json:"license,omitempty"`
```

Comment:
I could see returning all of these being helpful (vs only one), or even constructing the Modelfile on the client side eventually. It would also remove the need for the `type` parameter above.

Reply:
ended up removing the type parameter and sending everything back.